home-assistant-voice.yaml

```diff
@@ -1141,14 +1141,13 @@ script:
     parameters:
       increase_hue: bool  # True: Increase hue / False: Decrease hue.
     then:
-      - delay: 16ms
```

Comment:
I would prefer if we keep this one! 

The dial can "tick" very quickly!
This is a way to debounce it.

With @kahrendt , we managed to crash the device by rotating too fast.

Reply:
By setting this to less than 16 ms, we're just making sure it updates at _every_ call of `loop()`. In general, it won't run _faster_ than 16 ms, anyway, because that's typically the frequency at which `loop()` is called. I played with it quite a bit and it doesn't cause any problems. 😇

Reply:
No no. This is another place. 
I am fine having the effect run at every loop. 
But this is the script activated every tick. 

In our tests this help improving the stability of the device when spinner very fast

Reply:
> But this is the script activated every tick.

I'm not following you -- LED effects are only updated by `loop()`. Ultimately this change doesn't do anything other than ensure the LEDs might be updated more quickly if `loop()` somehow happens sooner than 16ms. That said, it's not a noticeable delay and I'm fine with keeping it as it was. 😄

Reply:
Oh wait, I was totally looking at the wrong thing. 🤦🏻‍♂️ Will fix...